package.json

```diff
@@ -10,10 +10,10 @@
     "packages/*"
   ],
   "resolutions": {
-    "@polkadot/api": "^0.82.0-beta.87",
-    "@polkadot/api-contract": "^0.82.0-beta.87",
+    "@polkadot/api": "^0.82.0-beta.94",
```

Comment:
Will bump the api now as well as soon as the metadata v7 PR passes. (Which will fix collective queries as well)